### PR TITLE
Fix URL in next post card for blog pages having date stamp

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

**Review by Isaiah-Hamilton — APPROVED** [review]

File: apps/www/pages/blog/[slug].tsx [hunk 1]
```diff
@@ -129,8 +131,12 @@ function BlogPostPage(props: any) {
 
   const NextCard = (props: any) => {
     const { post, label, className } = props
+
+    // [Joshen] I don't necessarily think this is the right fix though
+    const formattedUrl = post.url.split('/').pop()
```
Comment (inian) [inline apps/www/pages/blog/[slug].tsx:136]: I think [our current code](https://github.com/supabase/supabase/blob/3d14567cff0f4486b347eb9fc68b04d31b5abb58/apps/www/lib/posts.tsx#L42) adds year month and day..so should we pop multiple times?

Or just change posts.tsx to stop add dates

Comment (MildTomato) [reply apps/www/pages/blog/[slug].tsx:136]: hang on, this has possibly changed since

Comment (MildTomato) [reply apps/www/pages/blog/[slug].tsx:136]: the post.url shouldn't have any date formatting in it now

Comment (joshenlim) [reply apps/www/pages/blog/[slug].tsx:136]: i tried removing the date appending in lib/posts.tsx but then it messes up the URLs of the blog cards

Comment (joshenlim) [reply apps/www/pages/blog/[slug].tsx:136]: > so should we pop multiple times?
just once will do cause it'll pop the last item in the array

Comment (MildTomato) [reply apps/www/pages/blog/[slug].tsx:136]: https://github.com/supabase/supabase/pull/8253

**Comment by MildTomato** [comment]
Might be best to use this now
https://github.com/supabase/supabase/pull/8253

**Pull request closed without merging**